### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by imprashast**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,33 +31,29 @@
...
-    "@unocss/autocomplete": "^0.58.5",
-    "@warp-ds/eslint-config": "1.0.5",
-    "cz-conventional-changelog": "^3.3.0",
-    "eslint": "8.57.0",
-    "prettier": "3.2.5",
-    "rollup": "^4.10.0",
-    "semantic-release": "^23.0.0",
-    "vitest": "^1.2.2"
-  },
-  "eslintConfig": {
```
Comment (BalbinaK) [inline package.json:59]: Why did you need to drop this extension and add a `eslint.config.js` instead?

Comment (imprashast) [reply package.json:59]: Moved from eslint 8.x to 9.x :)

**Review by BalbinaK — APPROVED** [review]
